geoMean.js

```diff
@@ -0,0 +1,15 @@
+// Napisz funkcję geoMean(), która zwróci średnią geometryczną podanej tablicy.
+
+function geoMean(array) {
+    let multiples = array[0];
```

Comment:
You could set it to 1 in the beginning and then use a normal for loop starting from the first element. But this is ok as well.